library/src/test/java/org/apache/hadoop/io/file/tfile/DTFileTest.java

```diff
@@ -70,7 +73,7 @@ public void setUp() throws IOException
 
     conf.setInt("tfile.fs.input.buffer.size", tconf.fsInputBufferSize);
     conf.setInt("tfile.fs.output.buffer.size", tconf.fsOutputBufferSize);
-    path = new Path("tmp/dtfile");
+    path = new Path(ROOT, "/tmp/dtfile");
```

Comment:
@chandnisingh other tests in tfile using same pattern. I used the same path for consistency. This code was copied from apache tfile implementation. Do you want all tests to use target directory for test data?